### ISSUE TITLE
Text and Multitext in Properties

### ISSUE THREAD

**Issue by chliuhq** [obsidianmd/obsidian-clipper#104]
**Describe the solution you'd like**

Can we stop the automatic addition of quotation marks when using Text and Multitext in Properties?

**Comment by kepano** [comment]
Can you explain why?

**Comment by chliuhq** [comment]
I'm not sure what the benefit of automatically adding quotes is. I only know that when I need to use Dataview to retrieve properties, it becomes difficult because of the quotes. Perhaps adding an optional button would be more suitable?

**Comment by kepano** [comment]
Can you be more specific, what is the issue you are encountering in Dataview?

**Comment by chliuhq** [comment]
When I imported a web page file into Obsidian, I created a collections properties to indicate which collection the current file belongs to. This collection name is usually a file name. For example, when I input collections: [[A]], it is actually imported into Obsidian as collections: "[[A]]". When I want to retrieve files in the A file where the collections properties is A, I use the following code: WHERE contains(collections, this.file.name), but this code is actually ineffective. I don't know how to represent "[[A]]", as it seems difficult.
